framework/source/class/qx/bom/client/EcmaScript.js

```diff
@@ -303,8 +303,36 @@ qx.Bootstrap.define("qx.bom.client.EcmaScript",
     getStringTrim : function() {
       return typeof String.prototype.trim === "function";
     },
-    
-    
+
+
+    /**
+     * Checks if 'BigInt' type is supported.
+     * @internal
+     * @ignore(BigInt)
+     * @return {Boolean} <code>true</code>, if BigInt is available.
+     */
+    getBigInt : function() {
+      return typeof BigInt != "undefined";
+    },
+
+
+    /**
+     * Checks if 'toLocaleString' is supported on the BigInt object and whether
+     * it actually works
+     * @internal
+     * @ignore(BigInt)
+     * @ignore(BigInt.prototype.toLocaleString)
+     * @return {Boolean} <code>true</code>, if the method is supported and
+     *   works at least rudimentary.
+     */
+    getBigIntToLocaleString : function()
+    {
+      return typeof BigInt != "undefined"                          // BigInt type supported...
```

Comment:
Please user the more exact ```!==``` comparison here. thank you.

Reply:
...which does not make a difference in this case, but O.K. I'll change that.